src/settings-ui/Settings.UI/SettingsXAML/Views/MouseWithoutBordersPage.xaml

```diff
@@ -186,11 +186,18 @@
                     </tkcontrols:SettingsCard>
                 </controls:SettingsGroup>
                 <controls:SettingsGroup x:Uid="MouseWithoutBorders_ServiceSettings" IsEnabled="{x:Bind ViewModel.CanToggleUseService, Mode=OneWay}">
-                    <tkcontrols:SettingsCard x:Uid="MouseWithoutBorders_UseService">
-                        <ToggleSwitch
-                            x:Uid="MouseWithoutBorders_UseService_ToggleSwitch"
-                            IsEnabled="{x:Bind ViewModel.IsEnabled, Mode=OneWay}"
-                            IsOn="{x:Bind ViewModel.UseService, Mode=TwoWay}" />
+                    <InfoBar
+                        x:Uid="GPO_SomeSettingsAreManaged"
```

Comment:
Can we use the one for the single setting and move it between settings card and the other info bars?

Did you revisit which of the other bars make sense to show if policy is configured?

Reply:
Hi, did not revisit 🤔 Guess I should.

Reply:
Did the single setting change :) And removed the start as admin info bar since it made no sense if you can't really change the setting. Thank you for the review!